benchmarks/simple.py

```diff
@@ -38,78 +51,53 @@ def loop(rc, reset_last_key=None):
         time.sleep(0.05)
 
 
-def timeit(rc, itterations=50000):
+def timeit(rc, num):
     """
     Time how long it take to run a number of set/get:s
     """
-    t0 = time.time()
-    for i in xrange(0, itterations):  # noqa
+    for i in range(0, num):  # noqa
         s = "foo{0}".format(i)
         rc.set(s, i)
         rc.get(s)
 
-    t1 = time.time() - t0
-    print("{0}k SET/GET operations took: {1} seconds... {2} operations per second".format((itterations / 1000) * 2, t1, (itterations / t1) * 2))
 
-
-def timeit_pipeline(rc, itterations=50000):
+def timeit_pipeline(rc, num):
     """
     Time how long it takes to run a number of set/get:s inside a cluster pipeline
     """
-    t0 = time.time()
-    for i in xrange(0, itterations):  # noqa
+    for i in range(0, num):  # noqa
         s = "foo{0}".format(i)
-
         p = rc.pipeline()
         p.set(s, i)
         p.get(s)
         p.execute()
 
-    t1 = time.time() - t0
-    print("{0}k SET/GET operations inside pipelines took: {1} seconds... {2} operations per second".format(
-        (itterations / 1000) * 2, t1, (itterations / t1) * 2)
-    )
-
 
 if __name__ == "__main__":
-    __docopt__ = """
-Usage:
-  simple [--host IP] [--port PORT] [--nocluster] [--timeit] [--pipeline] [--resetlastkey] [-h] [--version]
-
-Options:
-  --nocluster        If flag is set then StrictRedis will be used instead of cluster lib
-  --host IP          Redis server to test against [default: 127.0.0.1]
-  --port PORT        Port on redis server [default: 7000]
-  --timeit           run a mini benchmark to test performance
-  --pipeline         Only usable with --timeit flag. Runs SET/GET inside pipelines.
-  --resetlastkey     reset __last__ key
-  -h --help          show this help and exit
-  -v --version       show version and exit
-    """
-
-    args = docopt(__docopt__, version="0.3.0")
-
-    startup_nodes = [{"host": args["--host"], "port": args["--port"]}]
-
+    args = docopt(__doc__, version="0.3.1")
```

Comment:
There is some minor lint issues within this main block. But i can correct that after i merge this.